### DISCUSSION TITLE
seqr container is unhealthy error

### DISCUSSION THREAD

**Discussion by xshah** [broadinstitute/seqr#3242] [Q&A]
Hi Seqr Team/Devs,

I am following the instructions for a local seqr install and running into this issue
~/seqr$ sudo docker-compose up -d seqr
seqr_redis_1 is up-to-date
Starting seqr_postgres_1 ... done
Starting elasticsearch   ... done

ERROR: for seqr  Container "992d1dd22e2d" is unhealthy.
ERROR: Encountered errors while bringing up the project.

Looking forward to any help if possible

**Comment by hanars** [comment]
Can you share the logs - `docker-compose logs seqr`

**Comment by xshah** [comment]
sure the logs command doesnt return anything.

sudo docker compose up seqr
[+] Running 3/0
 ⠿ Container elasticsearch            Created                                                                                                                                                        0.0s
 ⠿ Container seqr_install-redis-1     Created                                                                                                                                                        0.0s
 ⠿ Container seqr_install-postgres-1  Created                                                                                                                                                        0.0s
Attaching to seqr_install-seqr-1
container for service "elasticsearch" is unhealthy


seqr-v1:~/seqr_install$ sudo docker compose up elasticsearch
[+] Running 1/0
 ⠿ Container elasticsearch  Created                                                                                                                                                                  0.0s
Attaching to elasticsearch
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:42,470Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "version[7.17.9], pid[7], build[default/docker/ef48222227ee6b9e70e502f0f0daa52435ee634d/2023-01-31T05:34:43.305517834Z], OS[Linux/5.10.0-21-cloud-amd64/amd64], JVM[Oracle Corporation/OpenJDK 64-Bit Server VM/19.0.2/19.0.2+7-44]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:42,476Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "JVM home [/usr/share/elasticsearch/jdk], using bundled JDK [true]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:42,477Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "JVM arguments [-Xshare:auto, -Des.networkaddress.cache.ttl=60, -Des.networkaddress.cache.negative.ttl=10, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -XX:+ShowCodeDetailsInExceptionMessages, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dio.netty.allocator.numDirectArenas=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Dlog4j2.formatMsgNoLookups=true, -Djava.locale.providers=SPI,COMPAT, --add-opens=java.base/java.io=ALL-UNNAMED, -Djava.security.manager=allow, -XX:+UseG1GC, -Djava.io.tmpdir=/tmp/elasticsearch-11387453044729834191, -XX:+HeapDumpOnOutOfMemoryError, -XX:+ExitOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m, -Des.cgroups.hierarchy.override=/, -Xms16055m, -Xmx16055m, -XX:MaxDirectMemorySize=8417968128, -XX:InitiatingHeapOccupancyPercent=30, -XX:G1ReservePercent=25, -Des.path.home=/usr/share/elasticsearch, -Des.path.conf=/usr/share/elasticsearch/config, -Des.distribution.flavor=default, -Des.distribution.type=docker, -Des.bundled_jdk=true]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,129Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [aggs-matrix-stats]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,129Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [analysis-common]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,129Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [constant-keyword]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,130Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [frozen-indices]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,130Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [ingest-common]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,130Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [ingest-geoip]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,131Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [ingest-user-agent]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,131Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [kibana]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,131Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [lang-expression]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,131Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [lang-mustache]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,132Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [lang-painless]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,132Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [legacy-geo]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,132Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [mapper-extras]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,133Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [mapper-version]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,133Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [parent-join]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,133Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [percolator]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,133Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [rank-eval]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,134Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [reindex]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,134Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [repositories-metering-api]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,135Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [repository-encrypted]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,135Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [repository-url]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,135Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [runtime-fields-common]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,136Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [search-business-rules]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,136Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [searchable-snapshots]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,136Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [snapshot-repo-test-kit]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,137Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [spatial]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,137Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [transform]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,137Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [transport-netty4]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,138Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [unsigned-long]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,138Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [vector-tile]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,139Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [vectors]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,139Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [wildcard]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,139Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-aggregate-metric]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,139Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-analytics]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,140Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-async]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,140Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-async-search]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,140Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-autoscaling]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,140Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-ccr]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,141Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-core]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,141Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-data-streams]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,141Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-deprecation]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,142Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-enrich]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,142Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-eql]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,142Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-fleet]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,142Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-graph]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,142Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-identity-provider]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,143Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-ilm]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,143Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-logstash]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,143Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-ml]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,144Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-monitoring]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,144Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-ql]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,144Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-rollup]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,144Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-security]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,145Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-shutdown]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,145Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-sql]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,145Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-stack]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,145Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-text-structure]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,145Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-voting-only-node]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,146Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "loaded module [x-pack-watcher]" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,146Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "no plugins loaded" }
elasticsearch  | {"type": "server", "timestamp": "2023-02-25T06:13:45,262Z", "level": "ERROR", "component": "o.e.b.ElasticsearchUncaughtExceptionHandler", "cluster.name": "docker-cluster", "node.name": "1876cf035669", "message": "uncaught exception in thread [main]", 
elasticsearch  | "stacktrace": ["org.elasticsearch.bootstrap.StartupException: ElasticsearchException[failed to bind service]; nested: AccessDeniedException[/usr/share/elasticsearch/data/nodes];",
elasticsearch  | "at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:173) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:160) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:77) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:112) ~[elasticsearch-cli-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.cli.Command.main(Command.java:77) ~[elasticsearch-cli-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:125) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:80) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "Caused by: org.elasticsearch.ElasticsearchException: failed to bind service",
elasticsearch  | "at org.elasticsearch.node.Node.<init>(Node.java:1088) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.node.Node.<init>(Node.java:309) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:234) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:234) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:434) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:169) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "... 6 more",
elasticsearch  | "Caused by: java.nio.file.AccessDeniedException: /usr/share/elasticsearch/data/nodes",
elasticsearch  | "at sun.nio.fs.UnixException.translateToIOException(UnixException.java:90) ~[?:?]",
elasticsearch  | "at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106) ~[?:?]",
elasticsearch  | "at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]",
elasticsearch  | "at sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:397) ~[?:?]",
elasticsearch  | "at java.nio.file.Files.createDirectory(Files.java:700) ~[?:?]",
elasticsearch  | "at java.nio.file.Files.createAndCheckIsDirectory(Files.java:807) ~[?:?]",
elasticsearch  | "at java.nio.file.Files.createDirectories(Files.java:793) ~[?:?]",
elasticsearch  | "at org.elasticsearch.env.NodeEnvironment.lambda$new$0(NodeEnvironment.java:300) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.env.NodeEnvironment$NodeLock.<init>(NodeEnvironment.java:224) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.env.NodeEnvironment.<init>(NodeEnvironment.java:298) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.node.Node.<init>(Node.java:429) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.node.Node.<init>(Node.java:309) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:234) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:234) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:434) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:169) ~[elasticsearch-7.17.9.jar:7.17.9]",
elasticsearch  | "... 6 more"] }
elasticsearch  | uncaught exception in thread [main]
elasticsearch  | ElasticsearchException[failed to bind service]; nested: AccessDeniedException[/usr/share/elasticsearch/data/nodes];
elasticsearch  | Likely root cause: java.nio.file.AccessDeniedException: /usr/share/elasticsearch/data/nodes
elasticsearch  |        at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:90)
elasticsearch  |        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
elasticsearch  |        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
elasticsearch  |        at java.base/sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:397)
elasticsearch  |        at java.base/java.nio.file.Files.createDirectory(Files.java:700)
elasticsearch  |        at java.base/java.nio.file.Files.createAndCheckIsDirectory(Files.java:807)
elasticsearch  |        at java.base/java.nio.file.Files.createDirectories(Files.java:793)
elasticsearch  |        at org.elasticsearch.env.NodeEnvironment.lambda$new$0(NodeEnvironment.java:300)
elasticsearch  |        at org.elasticsearch.env.NodeEnvironment$NodeLock.<init>(NodeEnvironment.java:224)
elasticsearch  |        at org.elasticsearch.env.NodeEnvironment.<init>(NodeEnvironment.java:298)
elasticsearch  |        at org.elasticsearch.node.Node.<init>(Node.java:429)
elasticsearch  |        at org.elasticsearch.node.Node.<init>(Node.java:309)
elasticsearch  |        at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:234)
elasticsearch  |        at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:234)
elasticsearch  |        at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:434)
elasticsearch  |        at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:169)
elasticsearch  |        at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:160)
elasticsearch  |        at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:77)
elasticsearch  |        at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:112)
elasticsearch  |        at org.elasticsearch.cli.Command.main(Command.java:77)
elasticsearch  |        at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:125)
elasticsearch  |        at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:80)
elasticsearch  | For complete error details, refer to the log at /usr/share/elasticsearch/logs/docker-cluster.log
elasticsearch exited with code 1

**Comment by hanars** [comment]
This is the same issue discussed in https://github.com/broadinstitute/seqr/discussions/2090#discussioncomment-2641837 . The commands that user ran that fixed the issue which should do the same for you
```
sudo chown -R 1000:1000 ./data/elasticsearch
```

**Reply by xshah** [reply]
thank you so much ! it works now
